lib/bootstrap_form/inputs/check_box.rb

```diff
@@ -26,6 +23,14 @@ def check_box_with_bootstrap(name, options={}, checked_value="1", unchecked_valu
 
       private
 
+      def check_box_options(name, options)
+        check_box_options = options.except(:class, :label, :label_class, :error_message, :help,
+                                           :inline, :hide_label, :skip_label, :wrapper, :wrapper_class, :switch)
+        check_box_options[:class] = check_box_classes(name, options)
+        check_box_options[:aria] = { required: true } if options[:required]
```

Comment:
It took me a minute to figure out why we might want to make a check box required. I assume it's for cases where you won't accept the form unless the users checks something like: "I read and agree with the terms."

Would you mind adding a test to `test/bootstrap_checkbox_test.rb` that shows that the aria attribute is added correctly? I think the code is fine, but we need the tests to ensure that we don't introduce regressions in the future.

Reply:
You are correct i added a test for the checkbox, and also added the required class to the checkbox label